setup.py

```diff
@@ -15,7 +15,7 @@
 
 
 def read_long_description(filename="README.md"):
-    with open(filename) as f:
+    with open(filename, "r", encoding="utf-8") as f:
```

Comment:
Please separate this out as a separate PR on its own. We'll cut a release without any other changes first.
The other changes can be in a separate PR.